pushnotifications/src/main/AndroidManifest.xml

```diff
@@ -15,14 +15,6 @@
                 <category android:name="${applicationId}" />
             </intent-filter>
         </receiver>
-        <service
```

Comment:
I'm assuming that this was an automatic (or otherwise intended) deletion?

Reply:
This is intended. I removed firebase job dispatcher. Android WorkManager is the new jobs handler.